changelog/unreleased/bugfix-spaces-action-margins

```diff
@@ -0,0 +1,6 @@
+Bugfix: Margin between "New Space" button and actions
+
+Added margin to "New Space" button to have same size as gap in "Personal" actions.
+
+https://github.com/owncloud/web/issues/9384
+https://github.com/owncloud/web/pull/9390
```

Comment:
i would argue that we don't add changesets if it has "no user value" so to say (same for refactoring of code), but since its a "getting started" ticket its fine with me

Reply:
That's fair. I have removed it.